lambda/query-activity-log/models.ts

```diff
@@ -0,0 +1,43 @@
+type UrnFdnSub = string;
+type ClientId = string;
+type SessionId = string;
+
+export interface UserData {
+  user_id: UrnFdnSub;
+  govuk_signin_journey_id: string;
+  session_id: SessionId;
+}
+
+export interface TxmaEvent {
+  event_id: string;
+  timestamp: number;
+  timestamp_formatted: string;
+  event_name: string;
+  client_id: ClientId;
+  user: UserData;
+}
+
+export interface Activity {
+  type: string;
+  client_id: string;
+  timestamp: number;
+}
+
+export interface ActivityLogEntry {
+  event_type: string;
+  session_id: string;
+  user_id: string;
+  timestamp: number;
+  activities: Activity[];
+  truncated: boolean;
+}
+
+export interface UserActivityLog {
+  TxmaEvent: TxmaEvent;
```

Comment:
type parameters should be lower case i think

Reply:
yup okay I will fix this here and we can add ticket for all the other lambdas as well .